usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -674,8 +674,6 @@ def load_file_c(submission_attributes, db_cursor, award_financial_frame):
     awards_touched = []
 
     # format award_financial_frame
-    float_cols = ["transaction_obligated_amou"]
-    award_financial_frame[float_cols] = award_financial_frame[float_cols].fillna(0)
```

Comment:
Why isn't this needed anymore?

Reply:
Previously this was used to backfill the `transaction_obligated_amou` values that were Null to be 0. Now that we won't be taking in any Null or 0 values there is no need to run this.